CK3toEU4/Source/Mappers/ReligionMapper/ReligionMapper.cpp

```diff
@@ -188,6 +200,7 @@ void mappers::ReligionMapper::importCK3Faith(const CK3::Faith& faith,
 	newReligion.allowedConversion = allowedConversion;
 	newReligion.country = country;
 	newReligion.countrySecondary = countrySecondary;
+	LOG(LogLevel::Debug) << countrySecondary;
```

Comment:
Drop this when ready.

Reply:
Well I still haven't found out why it doesn't appear. Currently the output is blank, which means we're doing something wrong.